src/plugins/intel_cpu/src/plugin.cpp

```diff
@@ -249,12 +241,23 @@ void Engine::apply_performance_hints(ov::AnyMap& config, const std::shared_ptr<o
 
         auto num_requests = config.find(ov::hint::num_requests.name());
         if (num_requests != config.end()) {  // arrived with config to the LoadNetwork (and thus higher pri)
-            auto val = InferenceEngine::PerfHintsConfig::CheckPerformanceHintRequestValue(num_requests->second.as<std::string>());
+            int val = -1;
+            try {
+                val = std::stoi(num_requests->second.as<std::string>());
```

Comment:
can we use `as<int>`?

Reply:
oh, I still follow legacy std::string method, let us ov::any method.

Reply:
done